src/index_runner/utils/set_up_indexes.py

```diff
@@ -1,134 +0,0 @@
-import requests
```

Comment:
This code lives in `elasticsearch_writer` now.

Reply:
This module is now only responsible for sending off the index names, alias names, and mappings into the "elasticsearch_updates" topic using the "init_index" key.

elasticsearch_writer will take care of the logic around finding/creating the index, alias, and mapping